cssApp.css

```diff
@@ -49,31 +49,45 @@ html, body {
 	color: #555555;
 }
 
+.background {
+	display: flex;
+	justify-content: center;
+	background-size: 100% 100%;
+	padding: 100px 50px 0px;
+}
+
 .container {
-	height: 100%;
-	padding: 50px;
-    margin: 50px;
-    background-color: #fdfdfd;
+	overflow-x: auto;
 }
 
 .daylist {
 	display: flex;
 	overflow-x: auto;
+	overflow-y: hidden;
 }
 .day {
 	display: flex;
 	flex-shrink: 0;
-	border: 1px solid #000000;
+	position: relative;
+	top: 40px;
 	height: 60px;
 	width: 120px;
 	padding: 20px;
 	user-select: none;
+	background-color: #fdfdfd;
+}
+.day:nth-child(n+2) {
```

Comment:
Could you explain me what it's the purpose of this?

Reply:
This selector matches every nth element specified in brackets where:
n is a counter;
number is an offset;
In this case we are counting every element starting from the second.

Reply:
Instead of that, it's better to style all days and then use :last-child or :first-child
It's much more readable this way.